### PR TITLE
feat: update FE to use new API

### PR DESCRIPTION
**Pull request by x48-crypto**

### CODE REVIEW

**Comment by yearn-integrations** [comment]
## Important changes detected  (payment related code)
- Found constant `API_ENV` in files app/containers/Vaults/saga.js (`-    process.env.API_ENV === 'development' ||`)  
- Found constant `NODE_ENV` in files app/containers/Vaults/saga.js (`-    process.env.NODE_ENV === 'development'`)

**Comment by x48-crypto** [comment]
cc @dudesahn, need to QA this...

**Comment by xgambitox** [comment]
sorting by name is not working. app/components/VaultHeader still using displayName

**Comment by xgambitox** [comment]
I think it would be good to have a detailed doc of the changes in the structure from the previous API, so we can check if some changes where missed in the app. Searching for 'displayName', noticed that `app/containers/Vaults/MigrationBannerSvg` still uses `displayName`.

File: app/components/Vault/index.js [hunk 1]
```diff
@@ -530,7 +529,7 @@ const Vault = (props) => {
   if (apyType === 'error') {
     apyTooltip = _.get(ApyErrorDescriptions, `[${apy.description}].tooltip`);
   } else if (vaultIsBackscratcher) {
-    const currentBoost = _.get(apy, 'data.currentBoost', 0).toFixed(2);
+    const currentBoost = _.get(apy, 'composite.current_boost', 0).toFixed(2);
```
Comment (xgambitox) [inline app/components/Vault/index.js:532]: could not find 'composite.current_boost' on new API

**Comment by yearn-integrations** [comment]
## Important changes detected  (payment related code)
- Found constant `API_ENV` in files app/containers/Vaults/saga.js (`-    process.env.API_ENV === 'development' ||`)  
- Found constant `NODE_ENV` in files app/containers/Vaults/saga.js (`-    process.env.NODE_ENV === 'development'`)

File: app/components/Vault/index.js [hunk 2]
```diff
@@ -540,54 +539,54 @@ const Vault = (props) => {
...
+              <td>{truncateApy(apy.composite.base_apr)}</td>
             </tr>
             <tr>
               <td>Boost</td>
               <td>{currentBoost}x</td>
             </tr>
             <tr>
               <td>Total APY</td>
-              <td>{truncateApy(apy.data.totalApy)}</td>
+              <td>{truncateApy(apy.gross_apr)}</td>
```
Comment (turtlemoji) [inline app/components/Vault/index.js:585]: is this ok @x48-crypto ? In the new PR we use 
`totalApy: newApy.composite.pool_apy ? newApy.composite.pool_apy : newApy.composite.totalApy`

**Pull request closed without merging**